07_web_endpoints/basic_web.py

```diff
@@ -184,20 +184,21 @@ def expensive_secret():
 # The `expensive-secret` endpoint URL will still be printed to the output when you `modal serve` or `modal deploy`,
 # along with a "🔑" emoji indicating that it is secured with proxy authentication.
 # If you head to that URL via the browser, you will get a
-# [`407 Proxy Authentication Required`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/407) error code in response.
+# [`401 Unauthorized`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401) error code in response.
 # You should also check the dashboard page for this app (at the URL printed at the very top of the `modal` command output)
 # so you can see that no containers were spun up to handle the request -- this authorization is handled entirely inside Modal's infrastructure.
 
 # You can trigger the web endpoint by [creating a Proxy Auth Token](https://modal.com/settings/proxy-auth-tokens)
-# and then including the base64-encoded string `{TOKEN_ID}:{TOKEN_SECRET}` in a
-# [Proxy-Authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Proxy-Authorization).
+# and then including the token ID and secret in the `Modal-Key` and `Modal-Secret` headers.
 
 # From the command line, that might look like
 
 # ```shell
 # export TOKEN_ID=wk-1234abcd
 # export TOKEN_SECRET=ws-1234abcd
-# curl https://your-workspace-name--expensive-secret.modal.run -H "Proxy-Authorization: Basic $(echo -n $TOKEN_ID:$TOKEN_SECRET | base64)"
```

Comment:
I found this still works. Is it for backwards compat or will be supported in the future as well?

Reply:
Yup, it's for backwards compat and will be removed eventually.